assets/styles/layouts/_main-navigation.scss

```diff
@@ -137,6 +141,11 @@
     min-height: $navbar-mobile-min-height;
   }
 
+  @include touch {
+    display: flex;
+    align-items: center;
+  }
+
```

Comment:
also could be 

```
  &:not(.is-active) {
      @include touch {
        display: flex;
        align-items: center;
     }
  }

```

when the navbar is active ie open display flex breaks the scroll , that's why i need this 
![CleanShot 2023-10-15 at 09 29 28@2x](https://github.com/kodadot/nft-gallery/assets/44554284/e2b40adf-7fa9-430c-bdc9-6fbcfb79008d)